examples/dummy_plugin/dummy_plugin/jobs.py

```diff
@@ -9,4 +10,21 @@ class Meta:
         name = "Dummy job, does nothing"
 
 
-jobs = (DummyJob,)
+class SingletonJob(Job):
+    interval = IntegerVar(default=4, description="The time in seconds to sleep.")
+
+    class Meta:
+        name = "Singleton Job"
+        description = "Singleton job, only runs one at a time. Try running several and see what happens."
+        singleton = True
+
+    def run(self, data, commit):
+        interval = data["interval"]
+        self.log_info(message=f"Sleeping for {interval} seconds")
+        time.sleep(interval)
+        self.log_success(obj=None)
+        self.status = "complete"
```

Comment:
`Job` doesn't normally have a `status` attribute. Maybe remove this line as unneeded?